### PR TITLE
add terms and conditions

### LINKED ISSUE #2493: Form - Add guidance for Terms and Conditions Agreement flows
Ideally we allow the user to just accept the terms and conditions without requiring the user to scroll through the content, but we should double check EULA rules.. might need to check w/ legal. Could be product line specific?

image

TO DO:
- [x] Double check legal rules
- [x] Add guidance to site
- [x] Add an example to site
- [x] Follow up with Chandan Tiwari who raised this issue

### PR DESCRIPTION
**Pull request by britt6612**
[Deploy Preview](https://deploy-preview-INSERT_PR_#_HERE--keen-mayer-a86c8b.netlify.app/)

#### What does this PR do?
adds terms and conditions portion for guidance on forms
#### Where should the reviewer start?
forms.mdx
#### What testing has been done on this PR?
site
In addition to the feature you are implementing, have you checked the following:

**General UX Checks**
- [ ] Small, medium, and large screen sizes
- [ ] Cross-browsers (FireFox, Chrome, and Safari)
- [ ] Light & dark modes
- [ ] All hyperlinks route properly

**Accessibility Checks**
- [ ] Keyboard interactions
- [ ] Screen reader experience
- [ ] Run WAVE accessibility plugin (Chrome)

**Code Quality Checks**
- [ ] Console is free of warnings and errors
- [ ] Passes E2E commit checks
- [ ] Visual snapshots are reasonable

#### How should this be manually tested?
site
#### What are the relevant issues?
closes #2493 
#### Should this PR be mentioned in Design System updates?
yes
#### Is this change backwards compatible or is it a breaking change?
backwards compatible

### CODE REVIEW

**Review by bezAdane — APPROVED** [review]
Thanks!!!!

File: aries-site/src/examples/templates/forms/SignUpExample.js [hunk 1]
```diff
@@ -205,7 +205,7 @@ export const SignUpExample = () => {
               type="password"
             />
           </FormField>
-          <FormField htmlFor="terms-and-privacy">
+          <FormField required htmlFor="terms-and-privacy">
```
Comment (taysea) [inline aries-site/src/examples/templates/forms/SignUpExample.js:208]:
```suggestion
          <FormField required htmlFor="terms-and-privacy" name="termsAndPrivacy">
```

File: aries-site/src/pages/templates/forms.mdx [hunk 2]
```diff
@@ -111,6 +111,9 @@ Visual feedback is important to instill confidence in a user after form submissi
 
 Never force the user to wait for post-submission processing to complete before they can continue using an application.
 
+### Terms and Conditions
+Users must actively check a box next to a statement that shows they're agreed to the Terms and Conditions when signing up for an account, and the legal agreement is a simple link in the text for easy access and quick reference.
```
Comment (taysea) [inline aries-site/src/pages/templates/forms.mdx:115]:
```suggestion
Users must actively check a box next to a statement that shows they have agreed to the Terms and Conditions when signing up for an account. For easy access and quick reference, the legal agreement is a simple link in the text.
```

File: aries-site/src/pages/templates/forms.mdx [hunk 3]
```diff
@@ -111,6 +111,9 @@ Visual feedback is important to instill confidence in a user after form submissi
 
 Never force the user to wait for post-submission processing to complete before they can continue using an application.
 
+### Terms and Conditions
```
Comment (taysea) [inline aries-site/src/pages/templates/forms.mdx:114]: We need to add this to the forms sections in the template structure file.

**Review by taysea — APPROVED** [review]

**Pull request merged**